TESTS/host_tests/rtc_calc_auto.py

```diff
@@ -53,6 +53,7 @@ class RTC_time_calc_test(BaseHostTest):
     year_id = 0
              
              
+    __result = None
```

Comment:
Although this will fix an `AttributeError`, it introduces further confusion. It's unnecessary to override `result()` method in `RTC_time_calc_test` class in the first place (the device side decides if the test suite passed or not). I suggest removing `result()` and `teardown()` methods from this class altogether.

https://github.com/ARMmbed/mbed-os/blob/c8325152744393bd7292eb9548f3a36b7bb7e904/TESTS/host_tests/rtc_calc_auto.py#L134-L138

This way the code gets more readable.

Reply:
Was not sure what was the intention with having result method. I'll remove it, as soon as test is finished

Reply:
fixed

Reply:
Side note:
From what I've seen this `result()` method gets called if DUT does not send `{{end;success}}` / `{{end;failure}}` which may happen e.g. on test suite timeout.

https://github.com/ARMmbed/htrun/blob/9648fd73930cfa86967833e85a0f2ef5efaf9cd5/mbed_host_tests/host_tests_runner/host_test_default.py#L467-L473

Reply:
Looks correct, we are getting timeouts today, so that might needs patching as well